src/main/java/fr/openent/competences/controllers/CompetenceNoteController.java

```diff
@@ -70,28 +70,6 @@ public CompetenceNoteController(EventBus eb) {
         devoirsService = new DefaultDevoirService(eb);
     }
 
```

Comment:
Delete Get("/competences/note") API route in CompetencesNoteController because it is not used anywhere in frontend.